### PR TITLE
Features/actor tile

### PR DESCRIPTION
**Pull request by marcin10lw**
Hi!
If You could do check ActorTile live and give a review I'd grateful :)

### CODE REVIEW

**Review by mikolajgraczyk — APPROVED** [review]
Everything looks fine for me!

File: src/GlobalStyle.js [hunk 1]
```diff
@@ -11,5 +11,6 @@ export const GlobalStyle = createGlobalStyle`
 
     body {
       font-family: 'Poppins', sans-serif;
+      background-color: ${({ theme }) => theme.color.whisper};
```
Comment (HelkaPL) [inline src/GlobalStyle.js:14]: I thought that was mercury ( `#E5E5E5` )...  but i'm not sure.

**Review by HelkaPL — APPROVED** [review]
Appvored, 

But can any one else chcek the rignt background color of Project?
`color ? #E5E5E5 : #F5F5FA`

**Comment by marcin10lw** [comment]
That's a good point

**Pull request merged**